### DISCUSSION TITLE
Sort

### DISCUSSION THREAD

**Discussion by Salento78** [microsoft-search/pnp-modern-search#3841] [General]
Hi, I'm trying to add sorting on a numeric column, but it seems the sorting isn't working 
Can someone help me?
Thank you.

**Comment by kasperbolarsen** [comment]
Please use the Discussions option, not the Issues option for this kind of questions as this is obviously not an issue with the web parts :-)

**Comment by kasperbolarsen** [comment]
We will need to know more, like:

- Is that numeric column a List column or a site column?
- Can you see the crawled property in the schema?
- How and to which managed property is that crawled property mapped?
- Can you search for that Managed Property in SP Editor or SharePoint Search Query tool?
